src/index.html

```diff
@@ -1,12 +1,409 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo logo">
+        <a href="#">
+          <img src="./images/logo.png" alt="moyo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              class="nav__link" 
+              href="#laptops" 
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+
+      <div class="catalog__field">
+
+        <article class="catalog__card card" data-qa="card">
+
+          <img 
+            class="card__img" 
+            src="./images/imac.jpeg" 
+            alt="imac"
+          >
+
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="card__rate">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+
+              <li class="stars__star"></li>
+
+              <li class="stars__star"></li>
+
+              <li class="stars__star"></li>
+
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="card__stats">Reviews: 5</p>
+          </div>
```

Comment:
card__stats >>>card__stars

Reply:
stats is meant statistics))

Reply:
@APetrinko wow, I didn't think about it, yee, I thoungh it about stars div, so, you're good here)